web/less/landing.less

```diff
@@ -42,8 +42,13 @@ a {
   display: flex;
   justify-content: space-around;
   padding: 0 58px;
-  overflow-y: scroll;
-
+  overflow: scroll;
+  scrollbar-width: none; /* Firefox */
+  -ms-overflow-style: none;  /* IE 10+ */
+  &::-webkit-scrollbar {
+    width: 0px;
+    background: transparent; /* Chrome/Safari/Webkit */
+    }
```

Comment:
Hate to be picky, but can you please remove the extra whitespace at the beginning of this line?